tasks/recover_engine.yml

```diff
@@ -5,6 +5,7 @@
           username: "{{ vars['dr_sites_' + dr_target_host + '_username'] }}"
           password: "{{ vars['dr_sites_' + dr_target_host + '_password'] }}"
           ca_file: "{{ vars['dr_sites_' + dr_target_host + '_ca_file'] }}"
+      ignore_errors: False
```

Comment:
ignore_erros: false is default, so why do you need it?

Reply:
the entire block has ignore_errors: True (see line 158 in the file) so only the action related to the storage domains should be false all the time